pkgs/applications/editors/vim/plugins/overrides.nix

```diff
@@ -590,19 +590,19 @@ in
 
   cord-nvim =
     let
-      version = "2024-07-19";
+      version = "2024-09-25";
       src = fetchFromGitHub {
         owner = "vyfor";
         repo = "cord.nvim";
-        rev = "cd97c25320fb0a672b11bcd95d8332bb3088ecce";
-        hash = "sha256-66NtKteM1mvHP5wAU4e9JbsF+bq91lmCDcTh/6RPhoo=";
+        rev = "a26b00d58c42174aadf975917b49cec67650545f";
+        hash = "sha256-jUxBvWnj0+axuw2SZ2zLzlhZS0tu+Bk8+wHtXENofkw=";
       };
       extension = if stdenv.isDarwin then "dylib" else "so";
       rustPackage = rustPlatform.buildRustPackage {
         pname = "cord.nvim-rust";
         inherit version src;
 
-        cargoHash = "sha256-6FYf4pHEPxvhKHHPmkjQ40zPxaiypnpDxF8kNH+h+tg=";
+        cargoHash = "sha256-sRqzgzvhyhLYp8I1E1RGelCRxEFWrhFeW8vtdgYZr6c=";
```

Comment:
Hash is ment to be: `sha256-RS+0iLndye4U5eOoHLmwI+yxKAWS3JGHDbOK5jxf14Y=`

Reply:
Latest shows: ```sha256-YlTmkyEo1ZsBd3fMMFpkWWfWt7CfUP1BI2G/G5UtUwg=``` for me. I'll update it, thanks for pointing it out.